### ISSUE TITLE
osmnx rate limit issue

### ISSUE THREAD

**Issue by finnharbeke** [marceloprates/prettymaps#109]
As discussed in [this issue](https://github.com/gboeing/osmnx/issues/832) of osmnx, there is a 60 second pause before every request due to a change in the response format of the overpass api:

```
Unrecognized server status: "Rate limit: 3"
Pausing 60 seconds before making HTTP POST request
```
(printed from osmnx log)

This leads to the plot being loaded very slowly with unnecessary one-minute pauses.

osmnx's developer [commented](https://github.com/gboeing/osmnx/issues/832#issuecomment-1146383552)

>  It is the result of a change to the Overpass status endpoint's response format, reported [here](https://github.com/drolbr/Overpass-API/issues/668). I shall fix and release a patch version of OSMnx to correct this.

and fixed the issue. But it seems that prettymaps is dependent on an older osmnx version, hence the still existing issue when using prettymaps.

**Workaround**

The workaround I found is configuring `osmnx` to disable rate limiting, such that the unrecognized server status (which leads to the 60 second pause) is never read:

`osmnx.utils.config(overpass_rate_limit=False)`

I'm not sure though if it is fine to "just do". I guess rate limiting has its purpose and disabling it defeats that. I'm writing this here to share with others, for my personal use it should be fine so probably also for others.

Maybe @marceloprates knows if that setting is okay to be preset in the prettymaps code, such that users don't have to deal with it.

**Comment by marceloprates** [comment]
Hi, the new version  [v1.0.0](https://github.com/marceloprates/prettymaps/releases/tag/v1.0.0) should fix this, since it now depends on osmnx==1.2.2

**Comment by finnharbeke** [comment]
Nice, thank you!
